src/main/resources/format-pages.txt

```diff
@@ -30,6 +30,21 @@ http://www.openmicroscopy.org/info/slidebook for details.\n
 \n
 .. _Intelligent Imaging Innovations: http://www.intelligent-imaging.com/
 
+[3i Slidebook 7]
+extensions = .sldy
+owner = `Intelligent Imaging Innovations`_
+developer = `Intelligent Imaging Innovations`_
+bsd = yes
+software = `Slidebook <https://www.intelligent-imaging.com/slidebook/>`_
+weHave = * a small number of datasets \n
```

Comment:
Possibly unrelated to this PR, can we make our sample files public?

Reply:
The sample files were submitted publicly on the PR so that should be no problem. Do you want that included with this PR or can that be handled as a follow up?

Reply:
A follow-up PR is absolutely fine. I just happened to notice this but it should neither block the new page nor the release.